packages/realm-server/tests/realm-server-test.ts

```diff
@@ -434,116 +434,6 @@ module.only('Realm Server', function (hooks) {
     });
   });
 
-  module('card GET request', function (_hooks) {
```

Comment:
This looked to me to be an old copy of [this module](https://github.com/cardstack/boxel/blob/1616598d78205ac57aeeeabb43214e28d5703d7f/packages/realm-server/tests/realm-server-test.ts#L138), hopefully I didn’t miss any differences!